.phpstorm.meta.php

```diff
@@ -7,44 +7,45 @@
-			'editor.editor_support'             => \AmpProject\AmpWP\Editor\EditorSupport::class,
-			'extra_theme_and_plugin_headers'    => \AmpProject\AmpWP\ExtraThemeAndPluginHeaders::class,
-			'injector'                          => \AmpProject\AmpWP\Infrastructure\Injector::class,
-			'mobile_redirection'                => \AmpProject\AmpWP\MobileRedirection::class,
-			'obsolete_block_attribute_remover'  => \AmpProject\AmpWP\ObsoleteBlockAttributeRemover::class,
-			'optimizer'                         => \AmpProject\AmpWP\Optimizer\OptimizerService::class,
-			'plugin_activation_notice'          => \AmpProject\AmpWP\Admin\PluginActivationNotice::class,
-			'plugin_registry'                   => \AmpProject\AmpWP\PluginRegistry::class,
-			'plugin_suppression'                => \AmpProject\AmpWP\PluginSuppression::class,
-			'reader_theme_loader'               => \AmpProject\AmpWP\ReaderThemeLoader::class,
-			'rest.options_controller'           => \AmpProject\AmpWP\OptionsRESTController::class,
-			'rest.validation_counts_controller' => \AmpProject\AmpWP\Validation\ValidationCountsRestController::class,
-			'server_timing'                     => \AmpProject\AmpWP\Instrumentation\ServerTiming::class,
-			'site_health_integration'           => \AmpProject\AmpWP\Admin\SiteHealth::class,
-			'validated_url_stylesheet_gc'       => \AmpProject\AmpWP\BackgroundTask\ValidatedUrlStylesheetDataGarbageCollection::class,
-			'url_validation_cron'               => \AmpProject\AmpWP\Validation\URLValidationCron::class,
-			'save_post_validation_event'        => \AmpProject\AmpWP\Validation\SavePostValidationEvent::class,
-			'background_task_deactivator'       => \AmpProject\AmpWP\BackgroundTask\BackgroundTaskDeactivator::class,
-			'paired_routing'                    => \AmpProject\AmpWP\PairedRouting::class,
-			'paired_url'                        => \AmpProject\AmpWP\PairedUrl::class,
+			'admin.analytics_menu'               => \AmpProject\AmpWP\Admin\AnalyticsOptionsSubmenu::class,
+			'admin.google_fonts'                 => \AmpProject\AmpWP\Admin\GoogleFonts::class,
+			'admin.onboarding_menu'              => \AmpProject\AmpWP\Admin\OnboardingWizardSubmenu::class,
+			'admin.onboarding_wizard'            => \AmpProject\AmpWP\Admin\OnboardingWizardSubmenuPage::class,
+			'admin.options_menu'                 => \AmpProject\AmpWP\Admin\OptionsMenu::class,
+			'admin.polyfills'                    => \AmpProject\AmpWP\Admin\Polyfills::class,
+			'admin.paired_browsing'              => \AmpProject\AmpWP\Admin\PairedBrowsing::class,
+			'admin.validation_counts'            => \AmpProject\AmpWP\Admin\ValidationCounts::class,
+			'admin.plugin_row_meta'              => \AmpProject\AmpWP\Admin\PluginRowMeta::class,
+			'amp_slug_customization_watcher'     => \AmpProject\AmpWP\AmpSlugCustomizationWatcher::class,
+			'css_transient_cache.ajax_handler'   => \AmpProject\AmpWP\Admin\ReenableCssTransientCachingAjaxAction::class,
```

Comment:
Could you also add the missing [services](https://github.com/ampproject/amp-wp/blob/d0d21373128847f86d8fb5435e057e0f92544c33/src/AmpWpPlugin.php#L79-L82) here that were added to `AmpProject\AmpWP\AmpWpPlugin::SERVICES` in #6063, por favor.

Reply:
Done in e292be390